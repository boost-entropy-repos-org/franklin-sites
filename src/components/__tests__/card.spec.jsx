import React from 'react';
import renderer from 'react-test-renderer';

import Card from '../card';

describe('Card component', () => {
  test('should render', () => {
    const component = renderer
      .create(
        <Card title="Title">
          <span>Some content</span>
        </Card>,
      )
      .toJSON();
    expect(component).toMatchSnapshot();
  });
});
