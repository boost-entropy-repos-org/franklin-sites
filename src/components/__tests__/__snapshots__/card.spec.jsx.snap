// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card component should render 1`] = `
<div
  className="card"
>
  <h3
    className="card__title"
  >
    Title
  </h3>
  <div
    className="card__content"
  >
    <span>
      Some content
    </span>
  </div>
</div>
`;
